french/sortie/trappe/trappe.md

```diff
@@ -0,0 +1,4 @@
+En inspectant la chambre avec attention vous remarquez un relief au niveau du sol, vous vous approchez et réalisez que c’est une trappe.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.